waku/waku_store_sync/transfer.nim

```diff
@@ -34,7 +35,7 @@ type SyncTransfer* = ref object of LPProtocol
   peerManager: PeerManager
 
   # Send IDs to reconciliation protocol for storage
-  idsTx: AsyncQueue[SyncID]
+  idsTx: AsyncQueue[(SyncID, uint16)]
```

Comment:
Not for this PR, but these `uint16` makes me think that we need a `type Shard* = uint16` somewhere.